docs/applications_general.py

```diff
@@ -37,10 +27,10 @@
 
 # Set up an array of N simulations, starting from a single input.
```

Comment:
To clarify: In the initial version of the script I posted, this comment was intended to indicate that these two commands converted a single simulation input to N identical simulation inputs. Substitutions could be performed by modifying the N data streams, or by using different commands (not shown) that produce N non-identical data streams.